baseline_configs/one_phase/procthor/ithor/ithor_one_phase_rgb_base.py

```diff
@@ -0,0 +1,102 @@
+from abc import ABC
```

Comment:
File seems essentially identical to
```
baseline_configs/one_phase/one_phase_rgb_base.py
```

Reply:
Yes. I had just converted all class methods to instance/object methods to be able to directly use an `__init__` with `kwargs` in the final experiment config, but since I was just using the default configuration, there's no need to keep all that code duplication 👍